File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int length =0 ;
        int length1 =0;
        int n = nums.size();
        for(int i =0; i<n;i++)
        {
            if(nums[i]==1)
            {
                length+=1;
            }
            if(nums[i]==0)
            {
                if(length > length1)
                {
                length1 = length;
                }
                length = 0;
            }
        }
        if(length > length1)length1 = length;
        return length1;
        
    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        ans =0
        left =0
        for right in range(len(nums)):
            if nums[right]==1:
                left+=1
            else:
                ans = max(ans,left)
                left =0
        ans = max(ans,left)
        return ans
        